### DISCUSSION TITLE
Sprout v1.2.0 Launches!

### DISCUSSION THREAD

**Discussion by AIGamesDeveloper** [Sprout-Inc/Sprout-Scratch#7] [Announcements]
Hello Sprout Users,

Today, Sprout v1.2.0 will release bringing new features such as 

- Blacklist improvements
- Time for all rooms
- Username limits

On the contrary, the HTML file isn't working and we instead had to embed the project. Now, it has auto-updates. Please note that this is temporary and this will be fixed ASAP.

Any feedback is greatly appreciated.

**Full Changelog**: https://github.com/Sprout-Inc/Sprout-Scratch/compare/v1.1.1...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Sprout-Inc/Sprout-Scratch/releases/tag/v1.2.0'>Sprout v1.2.0 Launches!</a>.</em>

**Comment by WhatDoWeDoNowFromScratch** [comment]
Yeah! Now using 1.2.0 instead of boring 1.1.1
